README.md

```diff
@@ -4,35 +4,82 @@ Cognite GraphQL Python Generator
 [![GitHub](https://img.shields.io/github/license/cognitedata/cognite-gql-pygen)](https://github.com/cognitedata/cognite-gql-pygen/blob/master/LICENSE)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
 
-This is the Cognite GraphQL Python Generator, `gqlpygen`, which is a CLI for automatically generating Python SDK code
-from `.graphql` to interact with Cognite Data Fusion's (CDF) Flexible Data Models (DM). It also supports other cases,
-such as generating `.graphql` from `pydantic` classes.
+This is the Cognite GraphQL Python Generator, `gqlpyden`. The purpose of this package is to help developers to
+work with Cognite Data Fusion's (CDF) Data Models (DM) in Python.
 
 **[DISCLAIMER!]** This project is in a highly experimental no guarantees are made for consistency between versions. The
 project may also become deprecated if the experimentation turns out to be a dead end.
 
 
-### DM Non-GraphQl API
+The core functionality is to provide a Python client that match a data model. This enables the developer the following
+benefits
 
-This package also contains `cognite.dm_clients` which is a simplified wrapper for non-GraphQL endpoints in DM API v3.
+* Client side validation of the data before writing it to CDF.
+* Autocompletion matching the data model in the integrated developer environment (IDE). This is important as it enables:
+  * Discoverability of data model through Python.
+  * Reduced typing errors in development.
+* Keeping the language domain specific for the developer. Instead of working with generic concepts such as instances,
+  nodes, edges, the developer can work with the concepts in the data model.
 
-See [dm_clients/README.md](cognite/dm_clients/README.md) for more details.
 
 ## Installation
 
 To install this package:
 ```bash
 $ pip install cognite-gql-pygen
 ```
+
 ## Usage
 
-Currently (v0.1.0), there is no functionality, just some demo functions:
+The goal of the package is to have representations of all the types in a given data model with API calls to *.list()*,
+*.create()*, *.delete()*, and *.retrieve()* individuals for each type.
+
+![image](https://user-images.githubusercontent.com/60234212/232307167-0379c4ec-3b7a-4978-bfe7-d059e917e63f.png)
+
+
+You can specify the data models either as a `.graphql` schema or a `pydantic` classes in a `.py` file. Then, you can
+use the CLI to automatically generate the other representation as well as the `client.py` file which creates
+the API and the convenience method get_[client_name]_client().
+
+To generate from a `.graphql` schema you use the following command.
+
+```bash
+pygen topython 'PATH_TO_SCHEMA' --name "Cine"
+```
+
+This will create a `schema.py` and a `client.py` file in the directory you are running the command.
+
+To generate from `schema.py` use the following command
 
-Type the following command for help.
 ```bash
-cogpygen --help
+pygen togql 'PATH_TO_FILE'
```

Comment:
Just thinking (outside of this PR' scope), `pygen` is not the greatest name for a CLI tool, and neither is `dm_clients` (as in `dm_clients schema render`). Any ideas for a common CLI tool name? For allthethings, like:
  * `tool_name togql PATH_TO_FILE`
  * `tool_name topy ...`
  * `tool_name to_cdf ...` or `tool_name publish`  or `tool_name upload` ?

Reply:
I am leaning towards leaving some of this functionalities to the already existing CLI, cdf. Then, this name, or coming up with a better name will be easier.

Reply:
I guess just `dm` could work. wdyt?

Reply:
I decided `dm` is a better name, so changed it. `dm togql`, `dm topython`, `dm publish`, dm upload all works better.